### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'valid')

### ISSUE THREAD

**Issue by BlackYuzia** [pixijs/pixijs#9481]
### Current Behavior

When you try `texture.destroy()` before use it, you will get error: `Uncaught TypeError: Cannot read properties of null (reading 'valid')`.

### Expected Behavior

Don't sure what should be, but, suppose, texture could be "destroyed" before using it. Anyway, you shouldn't get error about `Cannot read properties of null`.

### Steps to Reproduce

Go to https://pixiplayground.com/#/edit/1JN3XJYaYlYSD4uLtAcog & open console.

### Environment

- **`pixi.js` version**: *7.2.4*
- **Browser & Version**: *Chrome 108*
- **OS & Version**: *Windows 10*
- **Running Example**:  https://pixiplayground.com/#/edit/1JN3XJYaYlYSD4uLtAcog

### Possible Solution

Replace `value.baseTexture.valid` by `value.texture?.valid` or `value.texture && value.texture.valid`.

### Additional Information

![image](https://github.com/pixijs/pixijs/assets/25010528/c4f0f978-125b-4ddc-9f88-a9cb1d58a2fe)

**Comment by miltoncandelero** [comment]
You destroyed the texture and then tried to use the destroyed texture?

I think an error is the logical result. 🤔

**Comment by BlackYuzia** [comment]
@miltoncandelero nope, I can use null as texture. But, if texture is destroyed, I get error about "null" instead of error by destroyed texture. That's not the same.

**Comment by miltoncandelero** [comment]
If pixi would just "not show anything" it would just hide your error and you will have to visually realize that a texture isn't showing on screen and then figure that it is because you destroyed the texture.

You shouldn't be using a destroyed object to begin with 🤷 . Pixi hiding the error of rendering a destroyed texture would be like wrapping your entire code in a try-catch block so that it never throws, but your program doesn't work anyway 🤷 

Fun fact: if you destroy a sprite and try to use it, you will get an error too. Destroyed objects are not to be used, they are destroyed.

At one point it was discussed about maybe checking if an object was already destroyed and throwing a more friendly error but it was decided that it is not pixi.js job to protect you from yourself 😅

**Comment by BlackYuzia** [comment]
@miltoncandelero that error is because `baseTexture` is null. That's all.
If we replace this by `baseTexture?.valid` this error will disapear.

**Comment by Zyie** [comment]
Hi there! 👋

We're doing some housekeeping and closing out older issues that haven't had recent activity.

PixiJS has evolved significantly since this issue was opened. If you're still experiencing this issue on PixiJS v8 (latest), please open a new issue with updated details and a minimal reproduction case.

Thanks for your understanding and for helping us keep the issue tracker focused and maintainable! 🩷
